cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1217,8 +1217,7 @@ spec:
                   nullable: true
                   properties:
                     maxBytes:
-                      type: integer
-                      minimum: 0
+                      type: string
```

Comment:
I'm not sure on this, but this might break existing users CRs which have an integer here, as K8S / OpenAPI wouldn't just convert / cast the "old" type to the new type as far as I know.

Reply:
let me do some research and see if there is a way to handle this

Reply:
I added a new field to the quotaSpec named maxBytesPretty for specifying the quota as a string. This is not ideal but it is safe.